src/offscreen/popupHandler.ts

```diff
@@ -34,6 +34,7 @@ export const openPopup = async (
   };
   const { id } = popupWithActions;
   popups[id] = popupWithActions;
+  // eslint-disable-next-line no-promise-executor-return
```

Comment:
I see this comment is added in many places. Maybe we can disable this rule globally?
@martinkaintas